mount/mount_windows.go

```diff
@@ -10,10 +15,61 @@ func (m *Mount) Mount(target string) error {
 	return ErrNotImplementOnWindows
```

Comment:
Question: why single-mount is unimplementable?
If unimplementable, please add a comment to the source code?

Reply:
I'll add a comment. This is due to the filesystem filter on Windows. When accessing a file in an underlying layer, there needs to be a target in the current layer, so a hardlink is created, pointing to the lower layer file (essentially). This means that without the lower layers, the current layer would be full of dead links. As a result, mounting a single layer would be almost unusable, so the only API exposed to mount layers on Windows requires all layers.